### PR TITLE
Added client name to data queue and error queue displays

### PR DESCRIPTION
**Pull request by PatrickWaweru**
Added client name to data queue and error queue displays

### CODE REVIEW

**Review by ojwanganto — APPROVED** [review]

File: api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java [hunk 1]
```diff
@@ -201,6 +204,35 @@ public String processIncomingRegistration(String resultPayload) {
...
+	 * Gets the client name from sent data
+	 * 
+	 * @param registrationNode the registration node
+	 * @param payload the payload data
+	 * @return the client name
+	 */
+	//private String getClientName(ObjectNode registrationNode, String payload) {
+	private String getClientName(ObjectNode registrationNode) {
+		String clientName = "";
+		//String givenName = JsonFormatUtils.readAsString(payload, "$['patient']['patient.given_name']");
```
Comment (makombe) [inline api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java:217]: Please remove the commented out code, I think they are not necessary

File: api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java [hunk 2]
```diff
@@ -201,6 +204,35 @@ public String processIncomingRegistration(String resultPayload) {
...
 	}
 	
+	/**
+	 * Gets the client name from sent data
+	 * 
+	 * @param registrationNode the registration node
+	 * @param payload the payload data
+	 * @return the client name
+	 */
+	//private String getClientName(ObjectNode registrationNode, String payload) {
```
Comment (makombe) [inline api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java:214]: Remove also this commented method

File: api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java [hunk 3]
```diff
@@ -45,6 +45,7 @@
 import org.openmrs.module.kenyaemr.metadata.HivMetadata;
 import org.openmrs.module.kenyaemr.util.EmrUtils;
 import org.openmrs.module.metadatadeploy.MetadataUtils;
+import org.openmrs.module.afyastat.utils.JsonFormatUtils;
```
Comment (makombe) [inline api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java:48]: The import is not used, please remove it.

File: omod/src/main/webapp/pages/afyastatHome.gsp [hunk 4]
```diff
@@ -420,11 +423,13 @@ tr:nth-child(even) {background-color: #f2f2f2;}
...
-                tr.append("<td>" + (displayRecords[i].clientName != "" ? displayRecords[i].clientName : displayRecords[i].patientUuid) + "</td>");
-            } else {
-                tr.append("<td>" + displayRecords[i].patientUuid + "</td>");
-            }
+            // if (tableId === 'queue') {
+            //     tr.append("<td>" + (displayRecords[i].clientName != "" ? displayRecords[i].clientName : displayRecords[i].patientUuid) + "</td>");
+            // } else {
+            //     tr.append("<td>" + displayRecords[i].patientUuid + "</td>");
+            // }
+            // tr.append("<td>" + displayRecords[i].patientUuid + "</td>");
```
Comment (makombe) [inline omod/src/main/webapp/pages/afyastatHome.gsp:431]: Do you think we will still need this?

**Review by makombe — APPROVED** [review]

File: api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java (hunk 1 above)
Comment (PatrickWaweru) [reply api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java:217]: Done

File: api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java (hunk 2 above)
Comment (PatrickWaweru) [reply api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java:214]: Done

File: api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java (hunk 3 above)
Comment (PatrickWaweru) [reply api/src/main/java/org/openmrs/module/afyastat/util/MedicDataExchange.java:48]: Done

File: omod/src/main/webapp/pages/afyastatHome.gsp (hunk 4 above)
Comment (PatrickWaweru) [reply omod/src/main/webapp/pages/afyastatHome.gsp:431]: Removed

**Pull request merged**